app/assets/javascripts/availability.js.coffee.erb

```diff
@@ -128,13 +133,21 @@ class AvailabilityUpdater
     display_request = location_services_element.attr('data-requestable')
     if availability_label_text not in available_non_requestable_labels
       display_request = 'true'
+    if availability_label_text == "Reserved for digital lending"
+      location_services_element.hide()
     if title_case(status) == 'On-site - in transit discharged'
```

Comment:
Can you remove() the element instead of only hiding? if you inspect the DOM using the hide() someone is still able to see the requests url.

Reply:
Sure; done.